tox.ini

```
@@ -37,6 +39,7 @@ deps =
     sphinx50: Sphinx>=5.0,<5.1
     sphinx51: Sphinx>=5.1,<5.2
     sphinx52: Sphinx>=5.2,<5.3
+    sphinx53: Sphinx>=5.3,<5.4
     sphinx60: Sphinx==6.0.0
```

```suggestion
    sphinx60: Sphinx>=6.0,<6.1
```
to match the others?

---

True - but it's also fixed in #1397 so I didn't want to duplicate the effort